rfcs/0017-tx-valid-since/0017-tx-valid-since.md

```diff
@@ -21,7 +21,7 @@ The highest 8 bits of `since` is `flags`, the remain `56` bits represent `value`
 * `flags & (1 << 7)` represent `relative_flag`.
 * `flags & (1 << 6)` and `flags & (1 << 5)` together represent `metric_flag`.
     * `since` use a block based lock-time if `metric_flag` is `00`, `value` can be explained as a block number or a relative block number.
-    * `since` use a epoch based lock-time if `metric_flag` is `01`, `value` can be explained as a epoch number or a relative epoch number.
+    * `since` use a epoch based lock-time if `metric_flag` is `01`, `value` can be explained as epoch information or relative epoch information.
```

Comment:
```suggestion
    * `since` use an epoch based lock-time if `metric_flag` is `01`, `value` can be explained as an absolute epoch or relative epoch.
```